build.sbt

```diff
@@ -224,6 +227,28 @@ lazy val http4s = project
   )
   .dependsOn(core)
 
+lazy val zioHttp = project
+  .in(file("adapters/zio-http"))
+  .settings(scalaVersion := scala13)
+  .settings(name := "caliban-zio-http")
+  .settings(commonSettings)
+  .settings(
+    resolvers ++= Seq(
+      "Sonatype OSS Snapshots" at "https://s01.oss.sonatype.org/content/repositories/snapshots"
+    ),
+    libraryDependencies ++= Seq(
+      "io.d11"           %% "zhttp"          % zioHttpVersion,
+      "io.circe"         %% "circe-parser"   % circeVersion,
+      compilerPlugin(
+        ("org.typelevel" %% "kind-projector" % "0.11.3")
+          .cross(CrossVersion.full)
+      ),
+      compilerPlugin("com.github.ghik" % "silencer-plugin" % silencerVersion cross CrossVersion.full),
```

Comment:
is it needed?

Reply:
Probably not, I think it's just me copy and pasting the http4s definition. I'll remove it!